src/python/grpcio/grpc/_cython/_cygrpc/server.pyx.pxi

```diff
@@ -61,16 +61,25 @@ cdef class Server:
           self.c_server, queue.c_completion_queue, NULL)
     self.registered_completion_queues.append(queue)
 
-  def start(self):
+  def start(self, backup_queue=True):
+    """Start the Cython gRPC Server.
+    
+    Args:
+      backup_queue: a bool indicates whether to spawn a backup completion
```

Comment:
We definitely don't want to expose this to the user. In fact, I think now is the time to pay down this debt and figure out whether or not a backup queue is actually needed. For my part, I can't actually think of a case where it *is* needed.